src/components/TextFieldComp.jsx

```diff
@@ -0,0 +1,22 @@
+import { FormControl, TextField } from '@mui/material';
+import { Box } from '@mui/system';
+import React from 'react';
+
+const TextFieldComp = () => {
+  const handleChange = () => {};
```

Comment:
suggestion: you keep consistency with the names, to just TextField, 
I understand why you have named, you can rename the one you are importing